src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -107,14 +150,42 @@ public IDisposable On(string methodName, Type[] parameterTypes, Func<object[], o
             return new Subscription(invocationHandler, invocationList);
         }
 
+        /// <summary>
+        /// Invokes a streaming hub method using the specified method name, return type and arguments.
+        /// </summary>
+        /// <param name="methodName">The hub method name.</param>
+        /// <param name="returnType">The hub return type.</param>
+        /// <param name="args">The hub arguments.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> to observe while waiting for the task to complete.</param>
+        /// <returns>
+        /// A <see cref="Task"/> that on completion indicates the hub method stream is open.
+        /// The value of the task is a <see cref="ChannelReader{T}"/> for the streamed hub method values.
```

Comment:
The value of the task is a <see cref="ChannelReader{T}"/> ~~for the~~ **which will yield** the streamed ~~hub method values~~ **results from the server**.